lib/blacklight.rb

```diff
@@ -39,7 +39,7 @@ def self.repository_class
     when /::/
       connection_config[:adapter].constantize
     else
-      Blacklight.const_get("#{connection_config[:adapter]}/Repository".classify)
```

Comment:
I meant, raise an exception ourselves that describes the problem and how to fix it (likely a misconfiguration or a Blacklight 5 -> 6 migration error?). If you think just a key error is sufficient, fine.